rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -250,6 +259,10 @@ def __init__(
             "label_ids" if self.component_config[INTENT_CLASSIFICATION] else "tag_ids"
         )
 
+    @staticmethod
+    def model_name():
+        return DIET
```

Comment:
Please add types. Shouldn't it be a string?

Reply:
Ah, so it is not the model name, but the model class, correct?

Reply:
yes, I wanted to ask you what is the best approach here, so that actual `RasaModel` type can be easily changed in subclass

Reply:
mmhh... not sure if that is needed at all

Reply:
renamed to `model_class()` for now

Reply:
We could also move the model creation to a separate method in train and load to a separate method and overwrite those in response selector. Not sure what is more elegant.

Reply:
what if we create class variable, like this one?

Reply:
Would not use class variables for that. I guess using `model_class` is the best option for now. We can update later, if we find a better solution. Will think about it until tomorrow.